src/main/java/cart/dto/CartItemQuantityUpdateRequest.java

```diff
@@ -1,6 +1,10 @@
 package cart.dto;
 
+import javax.validation.constraints.NotNull;
+
 public class CartItemQuantityUpdateRequest {
+
+    @NotNull(message = "수량을 입력해 주세요. 입력값 : ${validatedValue}")
     private int quantity;
```

Comment:
```suggestion
    private Integer quantity;
```
원시값이면 null이 될 수 없으니 `@Notnull` 어노테이션이 에러나지 않을까요?
래퍼 클래스로 바꾸는 거 어떤가요?! 🤔

Reply:
반영 완료 ><